app/src/main/java/com/univoice/feature/signup/StudentIdInputActivity.kt

```diff
@@ -26,6 +26,10 @@ class StudentIdInputActivity :
         disableButton()
         setupTextViews()
         setupNextButton()
+
+        binding.spStudentIdInput.post {
+            binding.spStudentIdInput.performClick()
+        }
```

Comment:
```suggestion
        binding.spStudentIdInput.apply {
            post {
                performClick()
            }
        }
        
```
P1: 이 로직을 initSpinnerFocus()같은 함수로 빼주세요